{
  "action": {
    "error": {
      "variety": [
        "Disposal error"
      ],
      "vector": [
        "Unknown"
      ]
    }
  },
  "actor": {
    "internal": {
      "motive": [
        "NA"
      ],
      "variety": [
        "End-user"
      ]
    }
  },
  "asset": {
    "assets": [
      {
        "variety": "M - Documents"
      }
    ],
    "cloud": [
      "Unknown"
    ]
  },
  "attribute": {
    "confidentiality": {
      "data": [
        {
          "variety": "Personal"
        }
      ],
      "data_disclosure": "Potentially",
      "data_victim": [
        "Customer"
      ],
      "state": [
        "Stored unencrypted"
      ]
    }
  },
  "discovery_method": {
    "unknown": true
  },
  "impact": {
    "overall_rating": "Unknown"
  },
  "incident_id": "926CBE10-A8AA-439D-8ECF-A5745E2E83D7",
  "plus": {
    "analysis_status": "First pass",
    "analyst": "blackfist",
    "attribute": {
      "confidentiality": {
        "credit_monitoring": "Unknown"
      }
    },
    "created": "2013-10-10T13:15:00Z",
    "github": "546",
    "master_id": "926CBE10-A8AA-439D-8ECF-A5745E2E83D7",
    "modified": "2014-05-10T00:58:19Z",
    "timeline": {
      "notification": {
        "day": 8,
        "month": 10,
        "year": 2013
      }
    }
  },
  "reference": "http://www.telecompaper.com/news/vodafone-germany-admits-to-another-data-breach--971750 (20131010);http://news.softpedia.com/news/Vodafone-Germany-Suffers-Another-Data-Breach-Customer-Information-Exposed-389950.shtml (20131010)",
  "schema_version": "1.3.6",
  "security_incident": "Confirmed",
  "source_id": "vcdb",
  "summary": "Temp employee throws customer data in recycling",
  "timeline": {
    "incident": {
      "month": 10,
      "year": 2013
    }
  },
  "victim": {
    "country": [
      "DE"
    ],
    "employee_count": "50001 to 100000",
    "government": [
      "NA"
    ],
    "industry": "51721",
    "region": [
      "150155"
    ],
    "victim_id": "Vodafone Group PLC"
  }
}